Sources/HummingbirdAuth/Authenticator/AuthRequestContext.swift

```diff
@@ -18,7 +18,7 @@ import NIOCore
 
 /// Protocol that all request contexts should conform to if they want to support
 /// authentication middleware
-public protocol AuthRequestContext {
+public protocol AuthRequestContext: RequestContext {
```

Comment:
This is a separate change really. I originally didn't conform AuthRequestContext to RequestContext because we had different context types for application and lambda, but now we use the context type we can add this back in.